contrib/babelfishpg_tsql/src/multidb.c

```diff
@@ -1277,12 +1277,6 @@ get_physical_schema_name_by_mode(char *db_name, const char *schema_name, Migrati
 
 			snprintf(result, (MAX_BBF_NAMEDATALEND), "%s_%s", db_name, name);
 		}
-		else if (!DbidIsValid(get_db_id(db_name)))
-		{
-			ereport(ERROR,
-					(errcode(ERRCODE_UNDEFINED_DATABASE),
-					 errmsg("database \"%s\" does not exist. Make sure that the name is entered correctly.", db_name)));
-		}
```

Comment:
So, this check throws an error message early and stops us from adding handling for these after this, Removing this will not affect the flow only the error message would differ. Instead of this error message we will get -> 
```~~ERROR (Message: TDS client library error: Msg #: 33557097, Msg state: 1, Msg: relation "dbo.fpn_table" does not exist, Server: BABELFISH, Process: , Line: 1, Level: 16)~~```

I have removed this check upon consulting with rishabh

Reply:
Have you tested single-db scenario from both TDS endpoint and PG endpoint with this change? For example, define this
```
CREATE DATABASE testdb
go

USE testdb
go

CREATE FUNCTION f_getval()
RETURNS INTEGER
AS
BEGIN
  RETURN 0
END;
go

CREATE PROCEDURE p_test
AS
    -- Call a UDF or a view under dbo schema
    SELECT  dbo.f_getval()
    RETURN 0
go
```
And, call the procedure from both PG endpoint and TDS endpoint

Reply:
output from both the endpoints
```
f_getval   
-----------
 0
```